lib/trento_web/views/session_view.ex

```diff
@@ -11,7 +11,8 @@ defmodule TrentoWeb.SessionView do
 
   def render("me.json", %{user: user}) do
     %{
-      username: user.username
+      username: user.username,
```

Comment:
I would obtain the `username` and `id` in the function signature. It looks prettier

Reply:
done